### ISSUE TITLE
polymophic associations

### ISSUE THREAD

**Issue by elfassy** [Exygy/minimongoid#28]
Is there a way to make a polymophic association? Something like: http://guides.rubyonrails.org/association_basics.html#polymorphic-associations

thanks

**Comment by kaptron** [comment]
Good idea, that's not currently something that's been implemented. If you want to take a stab at adding it to the codebase yourself that would be great, I'd be happy to merge a pull request -- otherwise I'll have to get back to you on implementing that once I've had some time to look into it. It seems like you have a totally usable workaround so hopefully that's good enough for now.

**Comment by elfassy** [comment]
ok i'll have a shot at it, cheers
